src/Workspaces/CSharp/Portable/Classification/Worker_Preprocesser.cs

```diff
@@ -324,6 +328,43 @@ private void ClassifyLoadDirective(LoadDirectiveTriviaSyntax node)
         ClassifyDirectiveTrivia(node);
     }
 
+    private void ClassifyIgnoredDirective(IgnoredDirectiveTriviaSyntax node)
+    {
+        AddClassification(node.HashToken, ClassificationTypeNames.PreprocessorKeyword);
+        AddClassification(node.ColonToken, ClassificationTypeNames.PreprocessorKeyword);
+
+        // The first part (separated by whitespace) of content is a "keyword", e.g., 'sdk' in '#:sdk Test'.
+        if (TryFindWhitespace(node.Content.ValueText, out var firstSpaceIndex))
+        {
+            var keywordSpan = new TextSpan(node.Content.SpanStart, firstSpaceIndex);
+            var stringLiteralSpan = new TextSpan(node.Content.SpanStart + firstSpaceIndex + 1, node.Content.Span.Length - firstSpaceIndex - 1);
```

Comment:
```suggestion
            var stringLiteralSpan = TextSpan.FromBound(node.Content.SpanStart + firstSpaceIndex + 1, node.Content.FullSpan.End);
```

Reply:
adding the `+1` doesn't seem necessary.  you're doing this to avoid classifying the spce.  but you could have `#:sdk   whatever`, and it would still classify the spaces after the first one.  So just split on the space index and make the math easier :)